qaframework-bdd-tests/src/test/resources/features/refapp-2.x/03-trackPatient/patient.feature

```diff
@@ -0,0 +1,100 @@
+Feature: Patient Management
+		
+  @selenium
+  @patient
+  Scenario Outline: Find Patient By Name
+  	Given a user creates a patient
+  	When a user clicks on Find Patient Record app from the home page
+  	And the system loads find patient page
+  	And a user enters the patient's "<name>"
+    Then the patient search is "<status>"
+    
+    Examples: 
+      | name  			| status       |
+      | rightName		| successful   |
+      | wrongName 	| unsuccessful |
+		
+  @selenium
+  @patient
+  Scenario: Find Patient Record
+  	Given a user creates a patient
+  	When a user clicks on Find Patient Record app from the home page
+  	And the system loads find patient page
+  	And a user searches for an existing patient by name
+    And a user clicks on first patient
+    Then the system loads the patient dashboard
+		
+  @selenium
+  @patient
+  Scenario: Register Unidentified Patient
+  	Given a user clicks on Register a patient App from the home page
+  	When the system loads Register a patient page
+  	And a user clicks on the Unidentified Patient checkbox
+    And a user selects a gender
+    And a user clicks Confirm button
+    Then the system adds patient record into the patients table
+    
+  @selenium
+  @patient
+  Scenario: Unidentified Patient Keyboard
```

Comment:
Whats the difference between this scenario and `Register Unidentified Patient`

Reply:
@mherman22 they are very closely related, the only difference is that the former is testing registration of unidentified patient and the later is testing if it is possible to access the key board of unidentified patient.